README.md

```diff
@@ -68,7 +68,7 @@ The numerous CLI enables app development.
 
 ### Building and running
 
-To build simply run `make build` without arguments, and the executable is stored
+To build simply run `make cli-build`, and the executable is stored
```

Comment:
Maybe we should include that you can do `cli-build` when you want use the production backend, and `cli-local` to build a version that connects to a local backend

Reply:
I think it is not relevant, as the server repo is private, and thus a local environment is not available to most people.